iOS/Inspector/Inspector.storyboard

```diff
@@ -346,7 +351,7 @@
                 </tableViewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="qKF-0N-vfa" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="864" y="58"/>
+            <point key="canvasLocation" x="863.768115942029" y="57.589285714285708"/>
```

Comment:
Could we revert this back so it’s integral (864, 58)?

Reply:
Storyboard is back to the previous commit.